### DISCUSSION TITLE
播放一会视频会崩溃

### DISCUSSION THREAD

**Discussion by AvoidGuest** [ywmoyue/biliuwp-lite#308] [Q&A] [answered]
不知道是不是我的问题视频放一会就会闪退卸载重装也没用

https://github.com/ywmoyue/biliuwp-lite/assets/107521025/c9fa5385-9422-4c49-b526-6f6a0886b3e5

**Comment by ywmoyue** [comment]
这个我感觉release包有问题，可能得clone下来代码用vs调试了

你先把日志通过邮件发给我看看吧，日志在设置帮助页最下面，邮箱地址在设置关于页

**Reply by AvoidGuest** [reply]
有几率的今天又不崩了，崩了再发吧。

**Comment by ywmoyue** [comment, marked as answer]
已收到日志，看日志像是请求版本信息失败导致的崩溃

建议尝试一下：
1. 在4.6.15版本中设置页关于页中手动点击检查更新看是否会崩溃
2. 更新4.6.16版本看看播放视频是否会崩溃

**Reply by AvoidGuest** [reply]
检查更新不会崩溃我试试4.6.16版，谢谢你的回复！

**Comment by ywmoyue** [comment]
@ZiLeYunk 请问还有出现过播放崩溃的现象吗，我这边使用的时候出现了，但我发现在崩溃时显卡占用有一瞬间满载，想问问你这边是否一样的情况

**Reply by AvoidGuest** [reply]
我这边没有出现了但是我隐约记得前几天崩溃一次不知道记错了没[doge]
